src/FEMMDeforLinearBaseModule.jl

```diff
@@ -92,6 +93,47 @@ function mass(self::FEMMDeforLinearAbstract,  geom::NodalField{FFlt},  u::NodalF
     return mass(self, assembler, geom, u);
 end
 
+function damping_ABC(self::FEMMDeforLinearAbstract, assembler::A,
+                     geom::NodalField{FFlt}, u::NodalField{T},
+                     impedance::T, surfacenormal::SurfaceNormal) where {A<:SysmatAssemblerBase, T<:Number}
+    fes = self.integdata.fes
+    nfes = count(fes);
+    DoFperNode = ndofs(u);
+    nNodesperElem = nodesperelem(fes);
+    sdim = ndofs(geom);
+    mdim = manifdim(fes);
+    enDoF = DoFperNode * nNodesperElem;
+    npts, Ns, gradNparams, w, pc  =  integrationdata(self.integdata);
+    startassembly!(assembler, enDoF, enDoF, nfes, u.nfreedofs, u.nfreedofs);
+    loc = zeros(FFlt, 1, sdim);
+    J = zeros(FFlt, sdim, mdim);
+    Ce = zeros(T, enDoF, enDoF);
+    Nn = zeros(FFlt, enDoF);
+    dofnums = zeros(FFlt, enDoF);
+    for i = 1:nfes
+        for j = 1:npts
+            locjac!(loc, J, geom.values, fes.conn[i], Ns[j], gradNparams[j])
```

Comment:
The matrix should be probably initialized to zero before we go through the loop over the integration points.